app.js

```diff
@@ -89,7 +89,7 @@ const styles = StyleSheet.create({
     backgroundColor: Platform.OS === 'ios' ? c.iosLightBackground : c.androidLightBackground,
   },
   navigationBar: {
-    backgroundColor: c.olevilleGold,
+    backgroundColor: c.tint,
```

Comment:
What if we change it to `c.olevilleGold`? It's a bit darker, but not as dark as the `tint`